spec/rubocop/cop/lint/useless_times_spec.rb

```diff
@@ -292,6 +292,15 @@ def my_method
 
         expect_no_corrections
       end
+
+      it 'registers no offense for 1.times without block' do
```

Comment:
It does register an offense, though?

Reply:
Yeah, right. Initially my intention was to not register something but I switched it up without updating this as well.

I'll try my hand at maybe making/extending some internal affair for this, there are already things that check for correct spec titles.